Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/DispatcherExtensions.cs

```diff
@@ -15,10 +16,12 @@ internal static class DispatcherExtensions
         /// Pushes an empty <see cref="DispatcherFrame"/> at <see cref="DispatcherPriority.ContextIdle"/>.
         /// </summary>
         /// <param name="dispatcher">A <see cref="Dispatcher"/> instance.</param>
+        /// <param name="priority">The <see cref="DispatcherPriority"/> of the idle frame.</param>
         /// <remarks>
         /// No frame is pushed if the <paramref name="dispatcher"/> is null, <see cref="Dispatcher.HasShutdownStarted"/> is true, or <see cref="Dispatcher.HasShutdownFinished"/> is true.
         /// </remarks>
-        internal static void DoEvents(this Dispatcher dispatcher)
+        [DebuggerStepThrough]
```

Comment:
Is this still necessary in the code?

Reply:
It's not really interesting code so when you're debugging it's annoying when you step into the function